crates/gosub_html5/tests/data/html5lib-tests/tokenizer/contentModelFlags.test

```diff
@@ -1,52 +1,5 @@
 {"tests": [
 
-{"description":"PLAINTEXT content model flag",
-"initialStates":["PLAINTEXT state"],
-"lastStartTag":"plaintext",
-"input":"<head>&body;",
-"output":[["Character", "<head>&body;"]]},
-
-{"description":"PLAINTEXT with seeming close tag",
-"initialStates":["PLAINTEXT state"],
-"lastStartTag":"plaintext",
-"input":"</plaintext>&body;",
-"output":[["Character", "</plaintext>&body;"]]},
-
-{"description":"End tag closing RCDATA or RAWTEXT",
-"initialStates":["RCDATA state", "RAWTEXT state"],
-"lastStartTag":"xmp",
-"input":"foo</xmp>",
-"output":[["Character", "foo"], ["EndTag", "xmp"]]},
-
-{"description":"End tag closing RCDATA or RAWTEXT (case-insensitivity)",
-"initialStates":["RCDATA state", "RAWTEXT state"],
-"lastStartTag":"xmp",
-"input":"foo</xMp>",
-"output":[["Character", "foo"], ["EndTag", "xmp"]]},
-
-{"description":"End tag closing RCDATA or RAWTEXT (ending with space)",
-"initialStates":["RCDATA state", "RAWTEXT state"],
-"lastStartTag":"xmp",
-"input":"foo</xmp ",
-"output":[["Character", "foo"]],
-"errors":[
-    { "code": "eof-in-tag", "line": 1, "col": 10 }
-]},
-
-{"description":"End tag closing RCDATA or RAWTEXT (ending with EOF)",
-"initialStates":["RCDATA state", "RAWTEXT state"],
-"lastStartTag":"xmp",
-"input":"foo</xmp",
-"output":[["Character", "foo</xmp"]]},
-
-{"description":"End tag closing RCDATA or RAWTEXT (ending with slash)",
-"initialStates":["RCDATA state", "RAWTEXT state"],
-"lastStartTag":"xmp",
-"input":"foo</xmp/",
-"output":[["Character", "foo"]],
-"errors":[
-    { "code": "eof-in-tag", "line": 1, "col": 10 }
-]},
```

Comment:
What happened to these test files?

Reply:
that should not be... I'll check what is going on..

Reply:
@emwalker I've added them back.. I accidentally committed edits in the tests and failed to restore them first